astropy/cosmology/_parameter/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from . import core, descriptors
+from .core import *
+from .descriptors import *
+
+__all__: list[str] = []
+__all__ += core.__all__
+__all__ += descriptors.__all__
```

Comment:
Why are these lines needed?
```suggestion

__all__: list[str] = []
__all__ += core.__all__
__all__ += descriptors.__all__
```

Reply:
I suppose they're not. I moved it to `cosmology.__init__`.